examples/shardingsphere-sample/shardingsphere-example-generator/src/main/java/org/apache/shardingsphere/example/generator/scenario/feature/type/EncryptExampleScenario.java

```diff
@@ -41,6 +44,18 @@
         return result;
     }
     
+    @Override
+    public Set<String> getJavaClassPathSet() {
+        return Collections.emptySet();
+    }
+    
+    @Override
+    public Set<String> getResourcePathSet() {
+        Set<String> result = new HashSet<>();
```

Comment:
Avoid to use Set as declared type, just use it in implementation.

Reply:
ok